dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
So I must be doing it wrong then, because, at least locally, the tests that test for messages at debug and trace levels fail if I don't do this. I'll look into it further.